### PR TITLE
[WIP] Working through examples to update commands

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/ROOT/pages/quickstart-functions.adoc [hunk 1]
```diff
@@ -40,36 +41,49 @@ $ docker login -u <username> -p <password>
...
 == Create a Functions project
 
-. Create a project directory:
+. Create a project directory, with a function and runtime:
 +
 
 [source,terminal]
 ----
-$ mkdir fn.example.io
+$ mkdir <directory_name> && faas init <function_name> -l node
```
Comment (lance) [inline modules/ROOT/pages/quickstart-functions.adoc:59]:
```suggestion
$ mkdir <directory_name> && faas init <directory_name> -l node
```
It is true that the directory name and function name are the same, but I think it's probably better to just use one in this case.

File: modules/ROOT/pages/quickstart-functions.adoc [hunk 2]
```diff
@@ -40,36 +41,49 @@ $ docker login -u <username> -p <password>
...
-. Navigate to the new directory:
+.Example output
+[source,termina]
+----
+Project path: /home/<user>/<directory_name>
+Function name: <function_name>
+Runtime: node
+Trigger: http
+----
+// Question: can the trigger be set to anything else?
```
Comment (lance) [inline modules/ROOT/pages/quickstart-functions.adoc:70]: By default the trigger is `http`, but you can use the `-t` flag when initializing to create a function that responds to CloudEvents. E.g.

```
faas init <directory_name> -l node -t events
```

File: modules/ROOT/pages/quickstart-functions.adoc [hunk 3]
```diff
@@ -40,36 +41,49 @@ $ docker login -u <username> -p <password>
...
+
+. Create the project files, build a container, and deploy the function as a Knative service:
 +
 
 [source,terminal]
 ----
-$ cd fn.example.io
+$ faas create -l node -c
```
Comment (lance) [inline modules/ROOT/pages/quickstart-functions.adoc:77]: This command is going away: https://github.com/boson-project/faas/issues/180

The `init` command you documented above will be renamed `create`, and there will be no create-build-deploy-all-in-one command. We will need to document `create` (previously `init`), `build` and `deploy` individually.

**Review by lance — COMMENTED** [review]
LGTM with a few minor comments. Sorry that things keep shifting. All should be settled in the next few days.

File: modules/ROOT/pages/quickstart-functions.adoc [hunk 4]
```diff
@@ -1,25 +1,57 @@
...
 * You have installed the https://docs.openshift.com/container-platform/4.5/cli_reference/openshift_cli/getting-started-cli.html#cli-getting-started[OpenShift `oc` CLI].
-* You have installed the https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-kn.html#installing-kn[Knative `kn` CLI].
 * You must have installed https://docs.docker.com/install/[Docker], and have access to a DockerHub account.
+* You have installed the https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-kn.html#installing-kn[Knative `kn` CLI]. Installing `kn` provides the binary that enables use of the `kn faas` commands. There is no additional setup required to use `kn faas` commands after `kn` is successfully installed.
```
Comment (lance) [inline modules/ROOT/pages/quickstart-functions.adoc:12]: @abrennan89 this is linking to the 4.5 CLI, but I believe `kn faas` is only available in 4.6.

Comment (abrennan89) [reply modules/ROOT/pages/quickstart-functions.adoc:12]: 4.6 docs are live now so I'll update this

**Pull request merged**